controller-server/src/main/java/com/yahoo/vespa/hosted/controller/proxy/ConfigServerRestExecutorImpl.java

```diff
@@ -86,25 +83,17 @@ public ProxyResponse handle(ProxyRequest proxyRequest) throws ProxyException {
         // TODO: Use config server VIP for all zones that have one
```

Comment:
Do you know why we do not use the configserver VIPs for non-aws zones?

Reply:
Primarily its because we use /zone/v2 in dashboard to access application config which is only available on the configserver to which the application was deployed (See VESPA-11845).